lib/temple/html/attribute_merger.rb

```diff
@@ -6,40 +6,35 @@ class AttributeMerger < Filter
       define_options merge_attrs: {'id' => '_', 'class' => ' '}
 
       def on_html_attrs(*attrs)
-        names = []
```

Comment:
just iterate on attributes

pretty sure 1.9.3 will return same order, if that is a concern